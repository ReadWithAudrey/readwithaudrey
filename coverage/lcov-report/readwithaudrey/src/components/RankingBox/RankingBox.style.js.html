<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for readwithaudrey/src/components/RankingBox/RankingBox.style.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../../prettify.css" />
    <link rel="stylesheet" href="../../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../../index.html">All files</a> / <a href="index.html">readwithaudrey/src/components/RankingBox</a> RankingBox.style.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>3/3</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>3/3</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">14x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">14x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">14x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import styled from 'styled-components'
&nbsp;
export const StyledRankingBox = styled.section.attrs({
  className: 'montserrat f5 lh-copy mb3',
})`
  color: #000;
  display: flex;
  a {
    font-family: color: #fff;
  }
`
&nbsp;
export const InputBox = styled.input.attrs({})`
  width: 23px;
  height: 23px;
  background: #fff;
  border: #d3d3d3 1px solid;
  text-align: center;
`
&nbsp;
export const Description = styled.span.attrs({
  className: 'ml1',
})`
  color: #000;
`
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Fri Oct 12 2018 12:51:26 GMT+0100 (British Summer Time)
</div>
</div>
<script src="../../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../../sorter.js"></script>
</body>
</html>
