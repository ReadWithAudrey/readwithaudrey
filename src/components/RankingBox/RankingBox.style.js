import styled from 'styled-components'

export const StyledRankingBox = styled.section.attrs({
  className: 'montserrat f6 lh-copy',
})`
  color: #000;
  a {
    font-family: color: #fff;
  }
`

export const InputBox = styled.p.attrs({
  className: 'fl',
})`
  width: 23px;
  height: 23px;
  background: #fff;
  border: #d3d3d3 1px solid;
`
