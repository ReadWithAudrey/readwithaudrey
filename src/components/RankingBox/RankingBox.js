import React from 'react'
import { StyledRankingBox, InputBox, Description } from './RankingBox.style'
import PropTypes from 'prop-types'

const RankingBox = props => {
  return (
    <StyledRankingBox>
      <InputBox placeholder={props.placeholder} />
      <Description>{props.children}</Description>
    </StyledRankingBox>
  )
}

RankingBox.propTypes = {
  children: PropTypes.string,
  placeholder: PropTypes.string,
}

export default RankingBox
