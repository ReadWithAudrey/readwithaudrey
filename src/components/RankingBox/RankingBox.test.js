import React from 'react'
import renderer from 'react-test-renderer'
import RankingBox from './RankingBox'

describe('RankingBox component snapshot test', () =>
  it('renders correctly', () => {
    const tree = renderer.create(<RankingBox />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
