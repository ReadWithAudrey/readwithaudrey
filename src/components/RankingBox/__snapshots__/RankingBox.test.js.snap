// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RankingBox component snapshot test renders correctly 1`] = `
<section
  className="sc-bdVaJa montserrat f6 lh-copy mb3 loVMeJ"
>
  <input
    className="sc-bwzfXH ATnwa"
  />
  <span
    className="sc-htpNat ml1 pt1 jkbruQ"
  />
</section>
`;
